tests/api/operation_logs/test_operation_logs_rest.py

```diff
@@ -17,29 +17,70 @@
 
 """Tests REST API operation logs."""
 
-import mock
+from flask import url_for
 from invenio_accounts.testutils import login_user_via_session
+from utils import get_json, postdata
 
-from rero_ils.modules.operation_logs.api import OperationLogsSearch
-from rero_ils.modules.operation_logs.models import OperationLogOperation
-
-
-def test_operation_log_entries(client, librarian_martigny, document):
-    """Test operation log entries after record update."""
-    with mock.patch(
-        'rero_ils.modules.operation_logs.listener.current_librarian',
-        librarian_martigny
-    ):
-        login_user_via_session(client, librarian_martigny.user)
-        print('_start_here')
-        document.update(
-            document, dbcommit=True, reindex=True)
-    search = OperationLogsSearch()
-    results = search.filter(
-        'term',
-        operation=OperationLogOperation.UPDATE).filter(
-        'term', record__pid=document.pid).filter(
-        'term', user_name=librarian_martigny.formatted_name
-    ).source().count()
-
-    assert results == 1
+
+def test_operation_logs_permissions(client, operation_log, patron_sion,
+                                    librarian_martigny, json_header):
+    """Test record retrieval."""
+    item_url = url_for('invenio_records_rest.oplg_item', pid_value='1')
+    item_list = url_for('invenio_records_rest.oplg_list')
+
+    res = client.get(item_url)
+    assert res.status_code == 404
+
+    res = client.get(item_list)
+    assert res.status_code == 401
+
+    res, _ = postdata(
+        client,
+        'invenio_records_rest.oplg_list',
+        {}
+    )
+    assert res.status_code == 401
+
+    res = client.put(
+        url_for('invenio_records_rest.oplg_item', pid_value='1'),
+        data={},
+        headers=json_header
+    )
+    assert res.status_code == 404
+
+    res = client.delete(item_url)
+    assert res.status_code == 404
+
+
+def test_operation_logs_rest(client, loan_pending_martigny,
+                             librarian_martigny,
+                             json_header):
+    """Test record retrieval."""
```

Comment:
Same docstring as in [L27][l27] of the same file.

[l27]: https://github.com/rero/rero-ils/pull/1937/files#diff-457510a69cc9d5953e587529bdcc0cc2e2e55b9e0fc7d609cfa82a530fd5cc12R27